### ISSUE TITLE
Broken interactivity

### ISSUE THREAD

**Issue by meggynw** [tcmg/adventure-issues#106]
**Game URL**
https://developer.chrome.com/devsummit/adventure/#cG9zaXRpb24veC8zMTcveS82OQ==
Coords = (317,69)

**Area**
devtools room; may have started in the fugu hallway

**Describe the bug**
I couldn't interact with any tables (or vending machines?). I could interact with NPC people and a few gifts. The frame rate was pretty low but manageable.
But when I reached (317,69), I tried to open a gift and there was a huge lag time. My ability to move around dropped drastically.
I closed the tab and waited a couple minutes. Then, reopened and resumed playing with a much better speed/interaction capability.

**To Reproduce**
After closing/reopening the tab, I haven't had the same experience.

**Desktop (please complete the following information):**
 - OS: macOS Catalina 10.15.7 19H2 (on a MacBook Pro 13in 2016)
 - Browser: Chrome
 - Version: 87.0.4280.67

**Comment by meggynw** [comment]
During buggy experience, FPS was about 3-4. Now I'm seeing ~36-42 FPS.

Also, during the bug, I think the dialog bubbles were stacking on top of each other? Or the old ones weren't disappearing. I'm now realizing how quickly they were supposed to dismiss.

**Comment by ChristofferBech** [comment]
Hi meggynw, we'll look into this. Do you by any chance remember if you used the teleport function to get there? We found a bug in that this morning, so I'm wondering if that caused it.

**Comment by meggynw** [comment]
No, I don't think I tried to use teleport until after I had closed/reopened the tab. At this point, I was still just exploring around :)
